tidb-cloud/import-csv-files.md

```diff
@@ -74,77 +74,77 @@ Because CSV files do not contain schema information, before importing data from
 
-        When **Custom Pattern** is enabled, you are required to specify a custom mapping rule between CSV files and a single target table in the following fields:
+        To modify the file pattern, click **Modify**, specify a custom mapping rule between CSV files and a single target table in the following fields, and then click **Scan**.
 
-        - **Object Name Pattern**: enter a pattern that matches the names of the CSV files to be imported. If you have one CSV file only, you can enter the filename here directly.
+        - **Source file name**: enter a pattern that matches the names of the CSV files to be imported. If you have one CSV file only, enter the file name here directly. Note that the names of the CSV files must include the suffix ".csv".
 
             For example:
 
             - `my-data?.csv`: all CSV files starting with `my-data` and one character (such as `my-data1.csv` and `my-data2.csv`) will be imported into the same target table.
             - `my-data*.csv`: all CSV files starting with `my-data` will be imported into the same target table.
 
-        - **Target Table Name**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
+        - **Target table name**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
+
+    - **Table Filter**: If you want to filter which tables to be imported, you can specify one or more table filters in this field, separated by `,`.
 
-4. Click **Import**.
+        For example:
 
-    A warning message about the database resource consumption is displayed.
+        - `db01.*`: all tables in the `db01` database will be imported.
+        - `db01.table01*,db01.table02*`: all tables starting with `table01` and `table02` in the `db01` database will be imported.
+        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
+        - `*.*` : all tables will be imported.
 
-5. Click **Confirm**.
+        For more information, see [table filter snytax](/table-filter.md#syntax).
+
+5. Click **Next**.
+
+6. On the **Preview** page, check and update the CSV specific configurations, including separator, delimiter, header, not-null, null, backslash-escape, and trim-last-separator. You can click the **Click here to edit csv configuration** link in the upper-right corner of the preview table to change the CSV configurations.
```

Comment:
```suggestion
6. On the **Preview** page, you can have a preview of the data, and click the **Click here to edit csv configuration** link to check and update the CSV-specific configurations, including separator, delimiter, header, not-null, null, backslash-escape, and trim-last-separator.
```

Reply:
The **Click here to edit csv configuration** link is quite obvious, so perhaps there is no need to highlight its location.